lab_5_scrapper/scrapper.py

```diff
@@ -195,6 +338,12 @@ def parse(self) -> Union[Article, bool, list]:
         Returns:
             Union[Article, bool, list]: Article instance
         """
+        response = make_request(self.full_url, self.config)
+        article_bs = BeautifulSoup(response.text, 'lxml')
```

Comment:
who guaranteed that the request is always success?

Reply:
fixed